Gemfile

```diff
@@ -1,29 +1,12 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-
-solidus_repo = ENV.fetch('SOLIDUS_REPO', 'solidusio/solidus')
-solidus_branch = ENV.fetch('SOLIDUS_BRANCH', 'master')
-solidus_i18n_repo = ENV.fetch('SOLIDUS_I18N_REPO', 'solidusio/solidus_i18n')
-solidus_i18n_branch = ENV.fetch('SOLIDUS_I18N_BRANCH', 'master')
-gem 'solidus_core', github: solidus_repo, branch: solidus_branch
-gem 'solidus_api', github: solidus_repo, branch: solidus_branch
-gem 'solidus_backend', github: solidus_repo, branch: solidus_branch
-gem 'solidus_sample', github: solidus_repo, branch: solidus_branch
-gem 'solidus_i18n', github: solidus_i18n_repo, branch: solidus_i18n_branch
 
 # Needed to help Bundler figure out how to resolve dependencies,
 # otherwise it takes forever to resolve them.
 # See https://github.com/bundler/bundler/issues/6677
 gem 'rails', '>0.a'
```

Comment:
`rails` is still needed by `bin/sandbox` to run the generator.

Reply:
I think we can remove the `'>0.a'` here. It was a trick needed for a scenario that is already solved.